src/fs/copy.js

```diff
@@ -1,5 +1,28 @@
+import fs from 'fs/promises';
+import { constants } from 'fs';
+
+const customErrMessage = 'FS operation failed';
+
 const copy = async () => {
-    // Write your code here 
+    try {
+        await fs.access('files', constants.F_OK);
+        const files = await fs.readdir('files');
+        
+        try {
+            await fs.access('files_copy', constants.F_OK);
+            throw Error(customErrMessage);
+        } catch (e) {
+            if (e.message !== customErrMessage) {
+                fs.mkdir('files_copy');
+                files.forEach(async (file) => {
+                    const content = await fs.readFile(`files/${file}`, 'utf8');
```

Comment:
I would suggest to use path module, as on different OS the file path can differ.

ex.

import path from "path";
import { fileURLToPath } from 'url';
const __filename = fileURLToPath(import.meta.url);
const __dirname = path.dirname(__filename);

const filePath = path.join(__dirname, 'files', file)